aiohttp/web_response.py

```diff
@@ -33,11 +34,11 @@ class ContentCoding(enum.Enum):
 ############################################################
 
 
-class StreamResponse(HeadersMixin):
+class StreamResponse(collections.MutableMapping, HeadersMixin):
 
     _length_check = True
 
-    def __init__(self, *, status=200, reason=None, headers=None):
+    def __init__(self, *, status=200, reason=None, headers=None, state=None):
```

Comment:
Drop `state` from accepted parameters.
Otherwise you should support it in all derived response classes like `web.Response`.
I think the second is pointless and don't worth a new param,